From 1904cbc9af2d518a77331deb082750366aaec90b Mon Sep 17 00:00:00 2001
From: aleks <aleks.stier@icloud.com>
Date: Tue, 31 May 2022 13:09:47 +0200
Subject: [PATCH] Use nord-theme

*"Inspired by the beauty of the arctic, the colors reflect the cold, yet
harmonious world of ice and the colorfulness of the Aurora Borealis."* -
[Nord Theme](https://www.nordtheme.com/)

The default behaviour of st is to reverse the fore- and background
colors of each selected cell. If you don't want the selection-colors to
be reveresed but instead have fixed fore- and background colors apply on
top of this patch the [selectioncolors](../selectioncolors/)-patch. Then
set the following settings in your config.h:

unsigned int selectionbg = 0;
unsigned int selectionfg = 256;
---
 config.def.h | 42 ++++++++++++++++++++----------------------
 1 file changed, 20 insertions(+), 22 deletions(-)

diff --git a/config.def.h b/config.def.h
index 91ab8ca..afd00fd 100644
--- a/config.def.h
+++ b/config.def.h
@@ -96,32 +96,30 @@ unsigned int tabspaces = 8;
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#3b4252", /* black   */
+	"#bf616a", /* red     */
+	"#a3be8c", /* green   */
+	"#ebcb8b", /* yellow  */
+	"#81a1c1", /* blue    */
+	"#b48ead", /* magenta */
+	"#88c0d0", /* cyan    */
+	"#e5e9f0", /* white   */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#4c566a", /* black   */
+	"#bf616a", /* red     */
+	"#a3be8c", /* green   */
+	"#ebcb8b", /* yellow  */
+	"#81a1c1", /* blue    */
+	"#b48ead", /* magenta */
+	"#8fbcbb", /* cyan    */
+	"#eceff4", /* white   */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
+	"#d8dee9", /* default foreground colour */
+	"#2e3440", /* default background colour */
 };
 
 
@@ -129,8 +127,8 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 258;
-unsigned int defaultbg = 259;
+unsigned int defaultfg = 256;
+unsigned int defaultbg = 257;
 unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
-- 
2.36.1

